GAE/src/org/waterforpeople/mapping/app/web/rest/DeviceGroupRestService.java

```diff
@@ -74,8 +74,7 @@
         DeviceGroup s = deviceGroupDao.getByKey(id);
```

Comment:
Its a small detail but referring to DeviceGroup as `s`  (in the `POST` and `PUT` methods) threw me off a little. I was expecting a survey when I saw it in line `marshallToDto(s)`